akka-persistence-query/src/main/scala/akka/persistence/query/PersistenceQuery.scala

```diff
@@ -23,6 +23,11 @@ object PersistenceQuery extends ExtensionId[PersistenceQuery] with ExtensionIdPr
    */
   override def get(system: ActorSystem): PersistenceQuery = super.get(system)
 
+  /**
+   * Java API.
+   */
+  override def get(system: ClassicActorSystemProvider): PersistenceQuery = super.get(system)
```

Comment:
That we have to add these to all the other extensions as well is a bit of a bummer but maybe not much worse than before.

Reply:
more places where this is needed? I would expect all extensions.
(and yes, that was what we had to do with the ordinary `get` also)

Reply:
Yep, exactly, needed everywhere. I'll work on that now.

Reply:
Done.